sql/database.go

```diff
@@ -498,6 +538,20 @@ func (tx *Tx) Exec(query string, encoder Encoder, decoder Decoder) (int, error)
 	return exec(tx.conn, query, encoder, decoder)
 }
 
+func fixError(err error) error {
```

Comment:
NIT:

```suggestion
func mapSqliteError(err error) error {
```

Reply:
Fixed. The original intent was only to check for `SQLITE_INTERRUPT` in this function which was happening after context cancellation, and thus such cancellations had wrong error returned from SQLite funcs, but later I made the func a bit more general, forgetting to update its name